R/zzz.R

```diff
@@ -12,28 +12,28 @@
       # Ideally we would source tools/installPythonClient.R to not
       # have to duplicate the synapseclient install code
       # system2(paste("Rscript ", getwd(), "/tools/installPythonClient.R ", getwd(), sep=""))
-      PYTHON_CLIENT_VERSION <- '4.0.0'
+      PYTHON_CLIENT_VERSION <- '4.4.0'
       # reticulate::virtualenv_create('r-reticulate')
       # reticulate::use_virtualenv('r-reticulate')
-      reticulate::py_install(c("requests<3", "pandas~=2.0.0", "pysftp", "jinja2", "markupsafe"))
+      reticulate::py_install(c("requests<3", "pandas==2.0.3", "pysftp", "jinja2", "markupsafe","numpy==1.24.4"))
       reticulate::py_install(c(paste("synapseclient==", PYTHON_CLIENT_VERSION, sep="")), pip=T)
       reticulate::py_run_string("import synapseclient")
     }
   )
-
+  
   reticulate::py_run_string(sprintf("synapserVersion = 'synapser/%s' ", utils::packageVersion("synapser")))
   reticulate::py_run_string("synapseclient.USER_AGENT['User-Agent'] = synapserVersion + ' '+ synapseclient.USER_AGENT['User-Agent']")
   reticulate::py_run_string("synapseclient.core.config.single_threaded = True")
-  reticulate::py_run_string("syn=synapseclient.Synapse(skip_checks=True)")
+  reticulate::py_run_string("syn=synapseclient.Synapse(skip_checks=True, debug=True)")
```

Comment:
During normal operations we would not want debug to be true. Could this we setup in such a way that it can be defined on the R side and passed in here? The idea being the default is `debug=False`, but can easily be toggled on?

Reply:
I like the idea of setting `debug=False`. It won't cause any confusions for the user but can be convenient for developer to toggle the mode on.